chia/data_layer/data_store.py

```diff
@@ -809,7 +834,7 @@ async def insert(
                 if side is not None:
                     raise Exception("Tree was empty so side must be unspecified, got: {side!r}")
 
-                await self._insert_root(
+                new_root = await self._insert_root(
```

Comment:
looks like we are never using this value i would remove

Reply:
Isn't this used for the `InsertResult` at the end in the case of inserting a first leaf into an empty tree?